frontend/locales/en/strings.json

```diff
@@ -583,6 +586,50 @@
       }
     }
   },
+  "users": {
+    "header": {
+      "title": "Users",
+      "description": "The list of users that have permission to access the AWS ParallelCluster User Interface."
+    },
+    "actions": {
+      "create": "Create user",
```

Comment:
thought: we should rely on global actions where possible. If I am not mistaken `refresh` and `delete` should be already defined.

Reply:
It's a good point. I thought about it and my conclusion was that we should be sure the two actions will always change copy "together". In general, re-using keys is a bit troublesome. We could maybe create a Component to re-use everywhere, thus we'll have only one key.

I propose we do it in future PRs